src/main/java/seedu/address/ui/PersonCard.java

```diff
@@ -15,6 +15,7 @@
 public class PersonCard extends UiPart<Region> {
 
     private static final String FXML = "PersonListCard.fxml";
+    private static final String TELEGRAM_PREFIX = "@";
```

Comment:
I like this. But i think we need to update the examples in user guide since users don't have to type @ now